tests/unittests/CPUDeviceManagerTest.cpp

```diff
@@ -295,7 +295,7 @@ TEST(CPUDeviceManagerTest, MultiModule) {
   updateInputPlaceholders(*ctx1, {module1->getPlaceholderByName("input")},
                           {&inputs});
 
-  std::unique_ptr<Context> ctx2 = std::make_unique<Context>(ctx1->clone());
+  std::unique_ptr<Context> ctx2 = std::make_unique<Context>();
```

Comment:
why change here?

Reply:
So by keying the tensor with a string we have to deal with duplicate keys, in this case there are two models in the test but all their placeholders have the same name. Creating ctx2 as a clone of ctx1 copies across Tensors for the placeholders in the first model, then when we allocate the second model's tensors we hit this collision (which is being handled via assert). Cloning the context was never an important part of this test anyway, so removing it doesn't affect the goal of the test.